src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/group_normalization.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "single_layer_tests/group_normalization.hpp"
+
+using namespace ov::test::subgraph;
+
+namespace {
+
+const std::vector<ov::test::ElementType> netPrecisions = {
+    ov::element::f32,
+};
+
+const std::vector<ov::test::InputShape> inputShapes = {
+    // static shape
+    {{3, 8, 2, 32}, {{3, 8, 2, 32}}},
```

Comment:
I don't see limitations for ranks in the [spec](https://github.com/openvinotoolkit/openvino/blob/e5c006ce74bfefff1e171bb476704dbc6d107988/docs/articles_en/documentation/openvino-ir-format/operation-sets/operations-specifications/normalization/group-normalization-12.rst). Could you please add the examples for 3D, 5D and 6D for example? 6D is needed to check case when rank of shapes is more than 6. And As I understand after decomposition shape rank of the node is equal to `original_rank + 1`. So if we take 6D shape, we will get case with 7D shape.

Reply:
have added input with more rank in test. Thanks!